packages/protocol/contracts/L1/libs/LibConstStrings.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+/// @title LibConstStrings
+/// @custom:security-contact security@taiko.xyz
+library LibConstStrings {
```

Comment:
If we use this name, it means all constants here are strings, so we may remove "_STR" from the var names.